app/components/forms/orders/order-form.js

```diff
@@ -37,8 +37,8 @@ export default Component.extend(FormMixin, {
   }),
   sameAsBuyer: false,
 
-  getRemainingTime: computed('data', function() {
-    let orderExpiryTime = this.get('data.event.orderExpiryTime');
+  getRemainingTime: computed('settings', function() {
```

Comment:
And what is this computer property returning ? I see no return statement at all.

Reply:
@niranjan94 , Same but it was implemented earlier this way, It's just used to calculate remaining time and this calls timer in it to calculate, But still no return it has

Reply:
@niranjan94 I tried to return getRemaining but alas it was of no use due to the re-rendering of a rendered value, That's why I think this.set is used to overcome this challenge

Reply:
This is very incorrect usage of a computed property. And a computed property should also never be reset by something else. Please refactor this correctly.